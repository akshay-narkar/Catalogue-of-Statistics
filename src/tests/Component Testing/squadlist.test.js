/*eslint-disable*/

import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Squadlist from '../../Components/squadlist';

const playerinfo = {
  id: 1,
  shirtNumber: 1,
  position: 'FWD',
  name: 'ABC',
  dateOfBirth: '12-02-90',
  nationality: 'Spanish',
  role: 'Player',
};

it('UI is unaltered wrt first snapshot', () => {
  const elem = renderer.create(<BrowserRouter><Squadlist playerinfo={playerinfo} /></BrowserRouter>).toJSON();

  expect(elem).toMatchSnapshot();
});
