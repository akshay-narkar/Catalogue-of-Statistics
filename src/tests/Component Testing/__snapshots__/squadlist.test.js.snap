// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
<div>
  <p
    className="ms-3 fw-light"
  >
    Name :
    <span>
       
    </span>
    <span
      className="fw-bold"
    >
      ABC
    </span>
  </p>
  <p
    className="ms-3 fw-light"
  >
    Position :
    <span>
       
    </span>
    <span
      className="fw-bold"
    >
       
      FWD
       
    </span>
  </p>
  <p
    className="ms-3 fw-light"
  >
    Nationality:
     
    <span>
       
    </span>
    <span
      className="fw-bold"
    >
       
      Spanish
    </span>
  </p>
  <p
    className="ms-3 fw-light"
  >
    Birth Date:
     
    <span>
       
    </span>
    <span
      className="fw-bold"
    >
       
      12-02-90
    </span>
  </p>
  <p
    className="ms-3 fw-light"
  >
    No:
     
    <span>
       
    </span>
    <span
      className="fw-bold"
    >
       
      1
    </span>
  </p>
  <p
    className="ms-3 fw-light"
  >
    Role:
     
    <span>
       
    </span>
    <span
      className="fw-bold"
    >
       
      Player
    </span>
  </p>
</div>
`;
